app/services/import/breweries.rb

```diff
@@ -0,0 +1,136 @@
+# frozen_string_literal: true
+
+module Import
+  class Breweries
+
+    attr_reader :log
+
+    def initialize
+      @log = ActiveSupport::Logger.new('log/import_breweries.log')
+      @dry_run = ENV['DRY_RUN'].present? ? ENV['DRY_RUN'].casecmp('true').zero? : false
+      @counter = { added: 0, failed: 0, skipped: 0, total: 0 }
+      @path_to_json = 'https://raw.githubusercontent.com/openbrewerydb/openbrewerydb/master/breweries.json'
+      @path_to_sql = 'https://raw.githubusercontent.com/openbrewerydb/openbrewerydb/master/breweries.sql'
+    end
+
+    def self.perform
+      new.perform
+    end
+
+    def perform
+      start_time = Time.now
+      puts "\nTask started at #{start_time}"
+      @log.info "Task started at #{start_time}"
+      
+      puts "\n!!!!! DRY RUN !!!!!\nNO DATA WILL BE IMPORTED\n" if @dry_run
+      
+      if ENV['TRUNCATE'].upcase == 'TRUE'
+        import_breweries_sql
+      else
+        puts "Updating breweries\n"
+        import_breweries
+      end
+      
+      output_summary
+      
+      end_time = Time.now
+      duration = (start_time - end_time).round(2).abs
+      puts "\nTask finished at #{end_time} and lasted #{duration} seconds."
+      @log.info "Task finished at #{end_time} and lasted #{duration} seconds."
+      @log.close
+    end
+
+    private
+
+    def import_breweries
+      puts "#{Time.now} : Getting raw breweries file - json".blue
+      connection = Faraday::Connection.new @path_to_json
+      response = connection.get(nil)

```

Comment:
Is `nil` necessary? Seems like `response = connection.get()` should do the same thing?

Reply:
No it's not necessary. Though it's strange that it's not requiring an argument for me now. When I've used Faraday previously, it's yelled at me for not having something as an argument.

Reply:
https://github.com/openbrewerydb/openbrewerydb-rails-api/pull/68/commits/9b5e26a749222b22247b172bd132448cd7aac6bf